packages/block-library/src/query-loop/edit.js

```diff
@@ -98,7 +98,18 @@ export default function QueryLoopEdit( {
 	);
 	const blockProps = useBlockProps();
 
-	if ( ! posts?.length ) {
+	if ( ! posts ) {
+		return (
+			<p { ...blockProps }>
+				{
+					// translators: %s: Name of Post Type (plural) that is loading e.g: "Posts".
+					sprintf( __( 'Loading %s…' ), postTypeName )
```

Comment:
Let's avoid the interpolation and the i18n problems it will cause. See explanation at https://github.com/WordPress/gutenberg/pull/13715#discussion_r255148472.

I think it's fine to go with either "Loading…", "Loading posts…", or something referencing the query rather than the posts. Honestly, the simpler, the better. :)

Reply:
Also, it's interesting to see the strings loading too slowly: when I load the editor with a Query block in it, I see "Loading Posts…" for a second and then it turns into "Loading Artigos…" (interpolation of pt_PT).

Reply:
Do you think the loading has something to do with the `Query` itself, or the translation handling in general?

Reply:
Totally fine to keep it simple

Reply:
> Do you think the slow loading of the translation has something to do with the `Query` itself, or the translation handling in general?

Translation in general